src/js/components/awardv2/idv/FundingSummary.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { formatMoneyWithPrecision } from '../../../helpers/moneyFormatter';
+
+const propTypes = {
+    totalTransactionObligatedAmount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    awardingAgencyCount: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+    federalAccountCount: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+};
+
+const FundingSummary = ({
+    totalTransactionObligatedAmount,
+    awardingAgencyCount,
+    federalAccountCount
+}) => (
+    <div className="award__col award-viz award-funding-summary">
+        <div className="award-viz__heading">
+            <div className="award-viz__icon">
+                <FontAwesomeIcon icon="chart-pie" />
+            </div>
+            <h3 className="award-viz__title">Federal Account Funding</h3>
+        </div>
+        <hr />
+        <div className="award-funding-summary__data">
+            <span>Total Funding Obligated</span>
+            <span>{typeof totalTransactionObligatedAmount === "number"
```

Comment:
This logic I would handle in the `updateSummaryData` in the `AwardMetaDataContainer` and set the state property `totalTransactionObligatedAmount` in the `AwardMetaDataContainer` initially to an empty string. This is only due to the fact that I like to handle data in the container when possible.